ScratchDjango/User/models.py

```diff
@@ -0,0 +1,41 @@
+import re
+
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError
+from django.db.models import CharField, EmailField
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from .managers import UserManager
+
+
+def check_email(email):
+    regex = r"\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,7}\b"
```

Comment:
Can we define this regex at a place from where it can be imported and used at other places

Reply:
we can create a file regex.py and define this regex as email_regex and import it whenever we need?

Reply:
Please do it.

Reply:
ok